modules/onboard-azure-eventhub/variables.tf

```diff
@@ -0,0 +1,175 @@
+variable "azure_eventhub_admin_email" {
+  description = "The email address to notify about the asset."
+  type        = string
+}
+
+variable "azure_eventhub_audit_pull_enabled" {
+  description = "If true, sonargateway will collect the audit logs for this system if it can."
+  type        = bool
+  default     = null
```

Comment:
Hasn't the default value for this one typically been `false`?

Reply:
Yes, but I left it as `null` here so that it would be treated as a computed value in the terraform state and instead was gonna just let connecting the related database assets connect this as well.

Reply:
On second thought, might actually be better to just hard-code as `null`...